packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -68,7 +68,8 @@ const AppConfig = ({children, locals = {}}) => {
             headers={headers}
             // Uncomment 'enablePWAKitPrivateClient' to use SLAS private client login flows.
             // Make sure to also enable useSLASPrivateClient in ssr.js when enabling this setting.
-            // enablePWAKitPrivateClient={true}
+            // TODO: Changed for testing private client. Restore config before merging.
+            enablePWAKitPrivateClient={true}
```

Comment:
You've already made a note here, but yes let's undo this change (and the one in ssr.js) before merging into develop.